### PR TITLE
Document records.yaml changes

### PR DESCRIPTION
**Pull request by brbzull0**

### CODE REVIEW

**Comment by brbzull0** [comment]
This is for https://github.com/apache/trafficserver/pull/9264

**Review by cmcfarlen — APPROVED** [review]

File: doc/release-notes/upgrading.en.rst [hunk 1]
```diff
@@ -130,6 +131,14 @@ The following :file:`records.yaml` changes have been made:
   have been removed.
 - The default values for ``proxy.config.http.request_header_max_size``, ``proxy.config.http.response_header_max_size``, and
   ``proxy.config.http.header_field_max_size`` have been changed to 32KB.
+- The records.yaml entry ``proxy.config.output.logfile`` have been renamed to ``proxy.config.output.logfile.name``.
```
Comment (cmcfarlen) [inline doc/release-notes/upgrading.en.rst:134]: `s/have/has/`

**Review by cmcfarlen — CHANGES_REQUESTED** [review]

**Review by cmcfarlen — APPROVED** [review]

**Comment by brbzull0** [comment]
I had to fix something I missed from the original [PR's](https://github.com/apache/trafficserver/pull/9264) doc.

**Review by cmcfarlen — DISMISSED** [review]

**Review by cmcfarlen — APPROVED** [review]

**Comment by cmcfarlen** [comment]
Cherry-picked to v10.0.x

**Pull request merged**